deebot_client/events/auto_empty.py

```diff
@@ -0,0 +1,22 @@
+"""Auto empty event module."""
+from dataclasses import dataclass
+
+from deebot_client.util import DisplayNameStrEnum
+
+from .base import Event
+
+
+class AutoEmptyMode(DisplayNameStrEnum):
+    """Enum class for all possible auto emptys."""
+
+    MODE_10 = "10"
```

Comment:
What is the meaning of these numbers?

Reply:
This are the number from the App for "minutes" or for "auto":

- The api sends them as string
  - for HA i implement it this way: [Deebot-4-Home-Assistant](https://github.com/MVladislav/Deebot-4-Home-Assistant/blob/8550109f268e12824cfea5050c4378c1a1b8e9fa/custom_components/deebot/translations/en.json#L79) 
- I also saw from [Full support for Deebot T20 Omni](https://github.com/DeebotUniverse/Deebot-4-Home-Assistant/issues/434) in file [Ecovacs T20 Dust bin auto empty.txt](https://github.com/DeebotUniverse/Deebot-4-Home-Assistant/files/12753381/Ecovacs.T20.Dust.bin.auto.empty.txt) that auto empty can also be "smart" for T20
  - do you want to have this directly included in this request? 

![auto empty](https://user-images.githubusercontent.com/7400017/281548524-d928a6b4-b334-43c2-827f-110047ce009b.png)

Reply:
Is it possible to set it also to 12 minutes (via an API call)?